test_rigs/unit-tests/configs/cisco_snmp

```diff
@@ -62,7 +62,9 @@ snmp-server enable traps vtp notifs
 snmp-server enable traps vtp vlancreate
 snmp-server enable traps vtp vlandelete
 snmp-server engineID local 000000090200000652D6C000
+snmp-server file-transfer access-group 8 protocol tftp
 snmp-server group cenic-nms v2c IPv4 auth
+snmp-server host 10.1.2.3 xyx
 snmp-server host 10.1.2.3 use-vrf management
```

Comment:
The line `snmp-server host 10.1.2.3 use-vrf management` is broken by your change above (see comment).